C/test/yahdlc_test.cpp

```diff
@@ -3,7 +3,7 @@
 #include <boost/test/unit_test.hpp>
 #include "yahdlc.h"
```

Comment:
Hmm. Else we should make the state available and have yahdlc_set_state(state)/yahdlc_get_state() functions and remove the *_with_state functions? Would make it a bit more clean.